nextflow_schema.json

```diff
@@ -97,6 +97,11 @@
                     "description": "Comma separated list of metadata column headers for statistics.",
                     "help_text": "Here columns in the metadata sheet can be chosen with groupings that are used for diversity indices and differential abundance analysis. By default, all suitable columns in the metadata sheet will be used if this option is not specified. Suitable are columns which are categorical (not numerical) and have multiple different values which are not all unique. For example:\n\n```bash\n--metadata_category \"treatment1,treatment2\"\n```\n\nPlease note the following requirements:\n\n1. Comma separated list enclosed in quotes\n2. May not contain whitespace characters\n3. Each comma separated term has to match exactly one column name in the metadata sheet"
                 },
+                "qiime_adonis_formula": {
+                    "type": "string",
+                    "description": "Formula for QIIME2 ADONIS metadata feature importance test for beta diversity distances",
+                    "help_text": "Here columns in the metadata sheet can be chosen that have no empty values, only unique values, or only identical values. Column headers need to be concatenated by `+` or `*`, e.g. `\"treatment+block\"`"
```

Comment:
One or two example formulae would be useful I think.

Reply:
Thankf for the suggestion! I agree and added examples and a link for further reading!